README.md

```diff
@@ -16,18 +16,19 @@ Install go dep
 go get -u github.com/golang/dep/cmd/dep
 ```
 
+Install atlas-db-controller for managing database resources:
+
+  * To create atlas-db custom resource and controller follow [link](https://github.com/infobloxopen/atlas-db/blob/master/README.md).
+  * To create database, tables and schema resources used by contacts app, modify `postgres.yaml` following specifications mentioned in [link](https://github.com/infobloxopen/atlas-db/blob/master/README.md) and run:
+  ```
+    kubectl create -f ./deploy/postgres.yaml
+  ```
+
 ### Local development setup
 
 Please note that you should have the following ports opened on you local workstation: `:8080 :8081 :9090 :5432`.
 If they are busy - please change them via corresponding parameters of `gateway` and `server` binaries or postgres container run.
 
-Run PostgresDB:
-
-``` sh
-docker run --name contacts-db -e POSTGRES_PASSWORD=postgres -e POSTGRES_USER=postgres -e POSTGRES_DB=contacts -p 5432:5432 -d postgres:9.4
-```
-Table creation in this example is omitted as it will be done automatically by `gorm`.
```

Comment:
I would not recommend removing this section as it describes how to run contacts app locally. In this case you will not be able to use atlas-db-controller. I would definitely change this section a bit saying that you need to create your tables manually.